website/docs/policy/weave-policy-profile.mdx

```diff
@@ -69,9 +68,10 @@ spec:
 To make use of this policy set in the profile:
 
 ```yaml
-config:
-  AGENT_ADMISSION_POLICY_SET: admission-policy-set # name of policy set to be used for admission
-  AGENT_AUDIT_POLICY_SET: audit-policy-set # name of policy set to be used for audit
+policy-agent:
```

Comment:
This works in the WGE chart context but perhaps not when configuring the profile for leaf cluster (not-namespaced), but I think thats pretty okay. Could add an [admonition](https://docusaurus.io/docs/markdown-features/admonitions) calling out the difference in usage?

Reply:
This should work in both situations since both WGE chart and the profile depend on the agent chart.

Reply:
Good point! 🎉